getting-started/setup-schemas-and-datasets.md

```diff
@@ -36,7 +36,7 @@ The following steps illustrate the creation of a sample schema for mobile data c
    1. From the left panel, select Schemas
    2. Select **Create schema**
    3. Select **XDM Experience event**
-   4. Under **Field Groups**, select **Add**; search and add the `Environment Details`,`Application Details`, and other pre-created or custom field groups as needed.
+   4. Under **Field Groups**, select **Add**; search and add the `Environment Details`,`Application Details`, `Adobe Experience Edge Autofilled Environment Details`, `AEP Mobile Lifecycle Details` and other pre-created or custom field groups as needed.
```

Comment:
I think we can remove the `Environment Details` and `Application Details` ones now since they are part of the other two, same for the screenshot.

Reply:
+1 to removing Environment Details and Application Details from instructions and image as they are part of the AEP Mobile Lifecycle Details field group.